src/router/pattern-router/router.ts

```diff
@@ -46,7 +48,7 @@ export class PatternRouter<T> implements Router<T> {
       if (routeMethod === method || routeMethod === METHOD_NAME_ALL) {
         const match = pattern.exec(path)
         if (match) {
-          handlers.push([handler, match.groups || Object.create(null)])
+          handlers.push([handler, match.groups || emptyParams])
```

Comment:
@EdamAme-x 

A quick question. Is there a possibility that the content of `emptyParams` will be changed? Like `emptyParams.someKey = "newValue"`.

Reply:
Since params should be abstracted by `c.req.param`, so I think there should be no such use case.

Reply:
Okay. Thanks!